cli/cli.go

```diff
@@ -40,7 +40,7 @@ func newCli(
 		} else {
 			cliOutput.Error(err.Error())
 
-			if re, ok := err.(*RunError); ok {
+			if re, ok := err.(RunError); ok {
```

Comment:
Just Curious was this the nil pointer bug we where seeing sometimes?

Reply:
Not here I just mentioned you where that was. The reason I removed the pointer from here was I just didn't see any reason for this being a pointer and leaving a potential for a NPE.